DNAnexus_to_igv/views.py

```diff
@@ -127,12 +131,12 @@ def nexus_search(request):
                     DNAnexus to find samples, please contact\
                     the bioinformatics team"""
                 )
-                error_log.error(re.sub(
+                logger.error(re.sub(
```

Comment:
this `re.sub()` seems unnecessary (was it me that wrote it? :eyes: )

I assume its just because the string is in triple quotes and split on lines so gets the tabs added, since it's in parentheses already for the logger just single quotes should work and not have tabs:

```
>>> print(
...     "Failed to load sample list from JSON, most "
...     "likely the JSON has not been generated with find_dx_data.py"
... )
Failed to load sample list from JSON, most likely the JSON has not been generated with find_dx_data.py
``` 

vs

```
>>> print(
...     """Failed to load sample list from JSON, most 
...     likely the JSON has not been generated with find_dx_data.py"""
... )
Failed to load sample list from JSON, most 
        likely the JSON has not been generated with find_dx_data.py
```

There's a few more too, can they just be changed to be simpler please